docs/en/wallet-instance-attestation.rst

```diff
@@ -2,56 +2,676 @@
+|                                   | for which the attestation is      |
+|                                   | being issued                      |
++-----------------------------------+-----------------------------------+
+| iat                               | Timestamp of issue of the         |
+|                                   | attestation                       |
++-----------------------------------+-----------------------------------+
+| exp                               | Attestation expiration timestamp. |
+|                                   | A good practice to avoid security |
+|                                   | problems is to have a limited     |
+|                                   | duration of the attestation.      |
++-----------------------------------+-----------------------------------+
+| type                              | String:                           |
+|                                   | "WalletInstanceAttestation"       |
++-----------------------------------+-----------------------------------+
+| policy_uri                        | Url to the privacy policy         |
+|                                   | of the wallet                     |
++-----------------------------------+-----------------------------------+
+| tos_uri                           | Url to the terms                  |
+|                                   | of use of the wallet provider     |
++-----------------------------------+-----------------------------------+
+| logo_uri                          | Logo url of the wallet provider   |
++-----------------------------------+-----------------------------------+
+| asc                               | Attested security context:        |
+|                                   | Represents a level of "trust" of  |
+|                                   | the service containing a Level Of |
+|                                   | Agreement defined in the metadata |
+|                                   | of the wallet provider            |
++-----------------------------------+-----------------------------------+
+| cnf                               | This parameter contains the JSON  |
+|                                   | object called jwk containing the  |
+|                                   | public key of the Wallet          |
+|                                   | Instance, necessary for the       |
+|                                   | holder binding                    |
++-----------------------------------+-----------------------------------+
+
+Signature
+^^^^^^^^^
+
+The JWT (Wallet Instance Attestation) thus composed is signed using the
+private key of the wallet provider present to which the backend wallet
+has access.
+
+Non-normative example
+^^^^^^^^^^^^^^^^^^^^^
+Below is an example of Wallet Instance Attestation:
+
+.. code-block:: javascript
+
+  {
+    "alg": "ES256",
+    "kid": "5t5YYpBhN-EgIEEI5iUzr6r0MR02LnVQ0OmekmNKcjY",
+    "trust_chain": [],
```

Comment:
please put here an example trust chain, produced using the python snippet here
https://docs.google.com/document/d/1uL61cfbFsOxC9zMJV81iTTUc7ZOv_WFgLD5Ruyr_fJ8/edit#heading=h.27g2dds1ic0m